dao/oracle.go

```diff
@@ -0,0 +1,91 @@
+package dao
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/emergency-auth-code-api/config"
+	"github.com/companieshouse/emergency-auth-code-api/models"
+
+	_ "github.com/godror/godror"
+)
+
+const QUERY = "select od.officer_id, od.officer_forename_1, od.officer_forename_2, od.officer_surname " +
```

Comment:
I don't think this needs exporting
```suggestion
const query = "select od.officer_id, od.officer_forename_1, od.officer_forename_2, od.officer_surname " +
```